features/page-lifecycle.yml

```
@@ -0,0 +1,7 @@
+name: Page Lifecycle
+description: "The Page Lifecycle API lets pages safely handle browser interventions such as page suspensions or discards from memory."
```

I think we have room enough to describe this rather completely:

```suggestion
description: "The page lifecycle API helps you safely handle page suspensions or discards from memory. The `freeze` and `resume` events fire when the browser suspends or resumes a page from memory while the `document.wasDiscarded` property reports whether the page was reloaded after being dropped."
```